azure-pipelines.yml

```diff
@@ -8,6 +8,17 @@ trigger:
 variables:
     DOTNET_VERSION: '2.2.207'
 
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily midnight build
+  branches:
+    include:
+      - dev
+      - master
+      - release/*
```

Comment:
So this change will make us running the release/* branch everynight. Do you suggest moving the previous releases into releases/ancient/* branches now?

Reply:
I set up the branch patterns using the best practices as recommended by AzDevops. We can change the daily runs to not target release branches.